cla_public/templates/macros/form.html

```diff
@@ -317,8 +317,11 @@
   {% if input_type == 'radio' %}
     <div class="govuk-radios">
     {% for option in field %}
+      {% if option.data in separated_options %}
+        <div class="govuk-radios__divider">or</div>
+      {% endif %}
       <div class="govuk-radios__item">
-        <input class="govuk-radios__input" id="{{option.id}}" type="radio" name="{{field.id}}" value="{{option.value}}" >
+        <input class="govuk-radios__input" id="{{option.id}}" type="radio" name="{{field.id}}" value="{{option.data}}" >
```

Comment:
Why the change from option.value to option.data and what's the difference

Reply:
option.value was an error introduced during my last change - so I fixed it here.  Should always have been option.data